actions/init/README.md

```diff
@@ -0,0 +1,28 @@
+# Wit Init Action
+
+This [GitHub Action](https://github.com/features/actions) will initialize a
+wit workspace for a repository with a wit-manifest.json. An environment
+package should be specified to describe the build environment. By default,
+environment-example-sifive is used to support reading a JSON description.
+
+See [action.yml](./action.yml) for a detailed list of input parameters.
+
+## Example Usage
+
+```yaml
+jobs:
+  test:
+    name: Scala compile
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Wit Init
+      uses: sifive/wit/actions/init@v0.13.2
+      with:
+        environment: git@github.com:sifive/environment-blockci-sifive.git::0.7.0
+
+    - name: Run wake scala compile
+      uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
+      with:
+        command: -x 'compileScalaModule myScalaModule | getPathResult'
```

Comment:
While I appreciate the more complete example, is this the place to have an action from another repo?

Reply:
I think anyone who uses this rule is going to want to use the two of them together. This comment is copy-pastable, to encourage people to do the right thing.

Reply:
This could be something generic,
```
- name: "run a command in the workspace"
  with:
    command: cowsay hello
```

Reply:
You say that wake has nothing to do with wit. In theory, that's true. In practice, if you are using wit, you are using wake. Someone who comes across this PR and/or docs is going to want to do both things.

Reply:
I've changed the documentation to include both examples.